### PR TITLE
Issue 023

### PR DESCRIPTION
**Pull request by murata2makoto**
Let me try again.

### CODE REVIEW

**Comment by murata2makoto** [comment]
Let me try again, as I mistakenly destroyed the commit on which the original pull request is based.

File: index.html [hunk 1]
```diff
@@ -56,7 +56,19 @@
             publisher: "W3C",
             date: "08 May 2019",
           },
-        },
+	  JEITA_IT-4002: {
```
Comment (himorin) [inline index.html:59]: Wrap by `'`.
Or, use `_` instead of `-`. (Syntax restriction of JavaScript on label)
